tests/unit/models/test_evc_deploy.py

```diff
@@ -1099,6 +1133,7 @@ def create_evc_intra_switch():
         uni_a.interface = interface_a
         uni_z.interface = interface_z
         attributes = {
+            "table_group": {"base": 0},
```

Comment:
Same here, let's try to use `evpl` and `epl`

Reply:
Changed